org-cyf-piscine/content/piscine/projects/music-data/index.md

```diff
@@ -0,0 +1,72 @@
+# Music data project
+
+One of the most common tasks in software is to take some data, process it in some way, and present it usefully to a user.
+
+For this project, we have supplied a collection of data for you to analyse. Your task is to write code which processes it and shows useful conclusions to a user.
+
+Your task is to make a small HTML + CSS + JavaScript frontend which displays the answers to several questions, which are listed below.
+
+Some principles you should remember throughout this project:
+1. This is a project about data processing, not UI. You should make a simple frontend to show the data, but you should spend as little time as you can on the frontend. No credit will be given for making prettier or more complicated frontend.
+2. You should assume the data will change in the future, but will have the same schema. So you shouldn't pre-compute anything - your code should always read the data and calculate results from scratch.
+
+TODO: Generate and link to `data.js`
+
+`data.js` is a file containing three functions:
+1. `getUserIDs()`: when called, returns an array of strings, each of which is a user ID.
+1. `getListens(userID)`: when called, returns an array of objects, each of which contains information about a single time that the given user listened to a song. The listens are sorted by when they happened, oldest to newest.
```

Comment:
I'm slightly on the fence about using a phrase like "listens": on one hand this is the sort of made-up word that we use all the time in practice, but it's not technically English.

So I suppose my point here is are we drawing the line such that we expect that (successful) piscine candidates will understand "jargon" like this?